packages/website/scripts/generate-kit-pages.js

```diff
@@ -50,7 +50,7 @@ ${markdown}`;
   try {
     const handle = await fs.opendir(kitPagesPath);
     console.info('kitgen: deleting existing kit pages');
-    rimraf(`${kitPagesPath}/*.md`, async (err) => {
+    rimraf(`${kitPagesPath}/*.md,${kitPagesPath}/*.mdx`, async (err) => {
```

Comment:
Is it worth leaving the `.md` files here? 🤔 It would clean up old remnant files in one's dev environment, that's true.

Reply:
Couldn't hurt. Then people don't have to worry about deleting the old files themselves.